### ISSUE TITLE
Empty page on Edge 18

### ISSUE THREAD

**Issue by kellda** [mozilla/send#1487]
When accessing "https://send.firefox.com/" with Edge 18 (and maybe other version), the page's content doesn't shows:

![screenshot](https://user-images.githubusercontent.com/59569234/86381526-bb7a8180-bc8d-11ea-8abe-4079271eb463.png)

The console says:

```
HTML1514: Une balise « <body> » supplémentaire a été trouvée. Seule une balise « <body> » doit être incluse par document.
send.firefox.com (77,7)

HTML1500: La balise ne peut pas se fermer automatiquement. Utilisez une balise de fermeture explicite.
send.firefox.com (91,17)

HTML1506: Le jeton est inattendu.
send.firefox.com (150,9)

SCRIPT1028: SCRIPT1028: Expected identifier, string or number
app.1ea9003f.js (23,163854)
```

These errors seems to be due to:
* a `noscript` tag between `head` and `body`, lines 62 to 75 in the index page
* `<use xlink:href="wordmark.d113c385.svg#logo" />`, line 91
* a `script` tag after `body`, lines 150 to 184 to the index page
* the syntax `c={`**`...t`**`,args:s,insideTermReference:!0}` not being supported by Edge

User agent :
Mozilla/5.0 (Windows NT 10.0; Win64; x64; ServiceUI 14) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36 Edge/18.18363

**Comment by kellda** [comment]
After some research, I fond that spread syntax in object literals is experimental and supported only from Edge 79 on.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Browser_compatibility
https://caniuse.com/#feat=mdn-javascript_operators_spread_spread_in_object_literals

**Comment by kellda** [comment]
This errors comes from Fluent, which is used in this project. The exact line is:
https://github.com/projectfluent/fluent.js/blob/fluent%400.13.0/fluent/src/resolver.js#L220

This syntax seems to be used quite a lot in Fluent

**Comment by dannycoates** [comment]
fixed in dd59605f3e67391cea52cc023f14ce61b85d6ebe
